tests/integration/components/reminder-item-test.js

```diff
@@ -0,0 +1,25 @@
+import { moduleForComponent, test } from 'ember-qunit';
```

Comment:
Make sure if you have a test file in your repo that it tests something legit - this is just a boilerplate test. Aka test SOMETHING about this component if you have this test in here.